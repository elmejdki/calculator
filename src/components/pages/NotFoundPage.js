import React from 'react';

const NotFoundPage = () => (
  <div className="flex center flex-column not-found max-height max-width">
    <h1>404</h1>
    <p>
      Page Not Found
    </p>
  </div>
);

export default NotFoundPage;
