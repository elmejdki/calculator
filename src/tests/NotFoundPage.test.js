import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from '../components/pages/NotFoundPage';

it('renders correctly the NotFound Page', () => {
  const tree = renderer.create(<NotFoundPage />).toJSON();

  expect(tree).toMatchSnapshot();
});
