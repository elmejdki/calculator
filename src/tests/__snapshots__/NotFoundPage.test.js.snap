// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the NotFound Page 1`] = `
<div
  className="flex center flex-column not-found max-height max-width"
>
  <h1>
    404
  </h1>
  <p>
    Page Not Found
  </p>
</div>
`;
